recipe/meta.yaml

```
@@ -21,26 +21,36 @@ build:
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir --no-build-isolation -vvv
 
 requirements:
+  build:
+    # we use go directly because we do not have activation feedstocks
+    # therefore we can't use compiler('go') and we skip linting
+    # for should_use_compilers
+    - go
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+
   host:
     - pip
-    - wheel
-    - python 
+    - python
+    - hatchling
     - setuptools
-    
+    - typing-extensions 4.7
+
   run:
     - click >=7.1,!=8.0.0
     - docker-pycreds >=0.4.0
     - gitpython >=1.0.0,!=3.1.29
-    - pathtools
-    - protobuf >=3.20.3,!=4.21.0,<5
+    - protobuf >=3.12.0,!=4.21.0,!=5.28.0,<6
```

```suggestion
    - protobuf >=3.19.0,!=4.21.0,!=5.28.0,<6
```
Because we do not support py<39, see https://github.com/wandb/wandb/blob/v0.19.1/pyproject.toml#L26-L27